src/main/800.c

```diff
@@ -26,7 +26,9 @@ INCLUDE_ASM("asm/main/nonmatchings/800", func_801B1094);
 
 INCLUDE_ASM("asm/main/nonmatchings/800", func_801B1144);
 
-INCLUDE_ASM("asm/main/nonmatchings/800", func_801B1298);
+u32 func_801B1298(s32 arg0) {
+    return (u32)(arg0 * 3) >> 1;
```

Comment:
Does it still match locally if you do...?

u32 func_801B1298(s32 arg0) {
    return (u32) (arg0 * 3) / 2;
}

Reply:
yes